extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -23,9 +23,13 @@ export const MicrosoftDatatoolsSchemaSqlSql = 'Microsoft.Data.Tools.Schema.Sql.S
 export const databaseSchemaProvider = 'DatabaseSchemaProvider';
 
 // Project Provider
-export const sqlDatabaseProjectTypeId = 'sqldbproj';
-export const projectTypeDisplayName = localize('projectTypeDisplayName', "SQL Database");
-export const projectTypeDescription = localize('projectTypeDescription', "Design, edit, and publish schemas for SQL databases");
+export const emptySqlDatabaseProjectTypeId = 'EmptySqlDbProj';
+export const emptyProjectTypeDisplayName = localize('projectTypeDisplayName', "SQL Database");
+export const emptyProjectTypeDescription = localize('projectTypeDescription', "Design, edit, and publish schemas for SQL databases from scratch");
```

Comment:
check with Drew before changing the description with this and for the Edge project description

Reply:
@dzsquared any ideas these descriptions?  I think we want strings that call out the difference between the one that starts from a blank state and the one that starts from a template, so I took a first pass.

Reply:
@Benjin 
How about?

SQL Database:
Design, edit, and publish schemas for SQL databases starting from an empty project
SQL Edge:  (drop Database please)
Design, edit, and publish schemas for SQL Edge starting from some common Edge objects


<alternatively>
SQL Database:
Develop and publish schemas for SQL databases starting from an empty project
SQL Edge:  (drop Database please)
Start with the core pieces to develop and publish schemas for SQL Edge


my strong opinions are:
- avoiding database in SQL Edge
- avoiding "from scratch" to be non-colloquial

Reply:
Noted!  I like the second option better, since it leads with the fact that it's seeding the project with core pieces.

Reply:
can you also update the screenshot in the PR description with the updated template name and descriptions?